### PR TITLE
broken link fix in multiple files

### PR DESCRIPTION
**Pull request by ronak-pal1**

### CODE REVIEW

**Comment by ronak-pal1** [comment]
Sorry, I forgot to sign my commits.

**Comment by ronak-pal1** [comment]
Can anyone review it?

File: docs/notebooks.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Everything you write in a notebook is a **cell**. A cell can contain text (in it
 ## Rich text
 
 Most cells have the ability to display rich text in them. You can **bold**,
-*italicize*, highlight,  ~~strikethrough~~ content or insert `in-line code` and [links](https://www.notion.so/fiberplane/docs.fiberplane.com). Cells include rich markdown support so you can type all of the above characters using their respective syntax:
+*italicize*, highlight,  ~~strikethrough~~ content or insert `in-line code` and [links](https://docs.fiberplane.com). Cells include rich markdown support so you can type all of the above characters using their respective syntax:
```
Comment (Nlea) [inline docs/notebooks.md:47]: Thanks for finding this. Could you include monospacing here so it shows the full [links](docs.fiberplane.com) path

**Comment by ronak-pal1** [comment]
> Dear @ronak-pal1 ,
> 
> Thank you for your contribution. Thank you for finding the broken image path and fixing it fixed.
> 
> The changes to the URL are not correct or needed. I added to comments in your PR
> 
> We also exclude the package json. Could you update your PR, excluding the package.json and including the other two comments to the URLs?
> 
> thank you

Thank you for highlighting my mistakes. I have remoted those commits but can you tell me how can I monospace in notebooks.md file as you said to do monospace. I have no idea.

**Comment by ronak-pal1** [comment]
> Dear @ronak-pal1 ,
> 
> Thank you for your contribution. Thank you for finding the broken image path and fixing it fixed.
> 
> The changes to the URL are not correct or needed. I added to comments in your PR
> 
> We also exclude the package json. Could you update your PR, excluding the package.json and including the other two comments to the URLs?
> 
> thank you

Thank you for highlighting my mistakes. I removed those commits and resolved the issue. Please check once.

**Comment by Nlea** [comment]
Hey, I am closing this because now the PR is empty without any changes

**Pull request closed without merging**